### PR TITLE
Add gitignore template

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Config"
+  ],
+  "name": "dotnet gitignore file",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Creates an empty gitignore file.",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json]: "Empty" should probably be changed here.

Comment (wli3) [reply template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json]: to "Creates an Empty gitignore file." ?

Comment (wli3) [reply template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json]: Maybe "Creates a gitignore file for a dotnet project".?

Comment (wli3) [reply template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json]: let's continue in email, cannot find all people in github

Comment (wli3) [reply template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json]: Changed to "Creates a gitignore file for a dotnet project.”

**Comment by wli3** [comment]
@seancpeters I edited my PR according to the email. And I don't have the merge access. Could you merge it if you think it is good?

**Comment by seancpeters** [comment]
Merged, thanks!

**Pull request merged**